src/Timer.cpp

```diff
@@ -0,0 +1,72 @@
+#include "../include/Timer.h"
+
+using namespace escape;
+using namespace std;
+
+/*
+* Constructor
+*/
+Timer::Timer() {
+
+	this->color		= sf::Color::White;
```

Comment:
use our spacing conventions.
Also, color should be handled in the view, not in our model classes